MANIFEST.in

```diff
@@ -1,2 +1,12 @@
-include LICENSE
+prune .github*
```

Comment:
I'm not very familiar with how MANIFEST.in works (the prune/exclude/include is confusing), but is the removal of the `include LICENSE` line here ok? I thought we wanted to keep the LICENSE file?

Reply:
Yepp. This is okay. As I mentioned here: https://github.com/icesat2py/icepyx/issues/170#issuecomment-773485313 By default, `setuptools_scm` includes everything in the repository, that's why the package size endded up being huge. To lessen that, we can utilize MANIFEST.in to exclude files. As stated:
> MANIFEST.in may still be used: anything defined there overrides the hook. This is mostly useful to exclude files tracked in your SCM from packages, although in principle it can be used to explicitly include non-tracked files too.

To understand the differences between prune/exclude/include etc. Take a look at: https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands :smile:

Reply:
Ok, so `setuptools_scm` includes everything in the repository by default, and then we use MANIFEST.in to exclude the stuff we don't need (i.e. as an override), gotcha!